homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -993,6 +993,11 @@ def close_tilt_position(self) -> ShadePosition:
         PowerViewShadeDualOverlappedFront,
         PowerViewShadeDualOverlappedRear,
     ),
+    11: (
```

Comment:
In a future PR it would be nice to make these named constants / enum so we don't have magic numbers here